data/json/uncraft/ammo/38super.json

```diff
@@ -6,7 +6,7 @@
     "difficulty": 5,
     "time": "5 s",
     "qualities": [ { "id": "PULL", "level": 1 } ],
-    "components": [ [ [ "lead", 2 ] ], [ [ "38super_casing", 1 ] ], [ [ "smpistol_primer", 1 ] ], [ [ "gunpowder_magnum_pistol", 3 ] ] ],
+    "components": [ [ [ "lead", 80 ] ], [ [ "38super_casing", 40 ] ], [ [ "smpistol_primer", 40 ] ], [ [ "gunpowder_magnum_pistol", 120 ] ] ],
```

Comment:
Here it's just expecting you to put it through the linter because the line has become too long with the numbers increasing. It's expecting 
```json
[
  {
    "result": "38super_fmj",
    "type": "uncraft",
    "skill_used": "gun",
    "difficulty": 5,
    "time": "5 s",
    "qualities": [ { "id": "PULL", "level": 1 } ],
    "components": [
      [ [ "lead", 80 ] ],
      [ [ "38super_casing", 40 ] ],
      [ [ "smpistol_primer", 40 ] ],
      [ [ "gunpowder_magnum_pistol", 120 ] ]
    ],
    "flags": [ "UNCRAFT_SINGLE_CHARGE" ]
  }
]
```

Reply:
Thanks for letting me know! I used your correction there and I'll make sure to follow any further errors when I wake up.